packages/connection-form/src/components/advanced-options-tabs/ssh-tunnel-tab/proxy-and-ssh-tunnel-tab.tsx

```diff
@@ -106,17 +113,33 @@ function ProxyAndSshTunnelTab({
   updateConnectionFormField,
   errors,
   connectionStringUrl,
+  _showProxySettingsForTesting,
 }: {
   errors: ConnectionFormError[];
   connectionStringUrl: ConnectionStringUrl;
   updateConnectionFormField: UpdateConnectionFormField;
   connectionOptions?: ConnectionOptions;
+  _showProxySettingsForTesting?: boolean;
```

Comment:
Nit, I'd suggest to fallback to using special testing only props on components as a total last resort: should be possible to just pre-configure the preferences in test to match the logic we're expecting here, unless I'm missing something 🤔

Reply:
Handled as a side-effect of the cleanup in  03fe506fae0c328d76e4220a01869f5954c62292 👍